crates/wasi-stub/src/lib.rs

```diff
@@ -254,6 +245,8 @@ pub fn stub_wasi_functions(
                 locals: locals.into_boxed_slice(),
                 expression: Expression {
                     instrs: instructions.into_boxed_slice(),
+                    branch_hints: Box::new([]),
+                    instr_spans: None,
```

Comment:
Only semantic difference. I'm assuming default values are ok. Didn't break downstream crate using it.